readthedocs/config/tests/test_config.py

```diff
@@ -978,7 +978,8 @@ def test_new_build_config(self):
         assert build.using_build_tools
         assert isinstance(build.build, BuildWithTools)
         assert build.build.os == 'ubuntu-20.04'
-        assert build.build.tools == {'python': '3.9'}
+        assert build.build.tools['python'].version == '3.9'
+        assert build.build.tools['python'].full_version == '3.9.7'
```

Comment:
Shouldn't these tests access the `settings.RTD_DOCKER_` variable? Otherwise, each time we change that setting, we will need to change the test as well.